docker/README.adoc

```diff
@@ -198,13 +198,20 @@ Python and Ansible are now also installed in the image, in case you need to exte
 [[build]]
 ## Build
 
-All images are based upon a `base` image which (in theory) should not have to be rebuilt too often. In order to build it:
+All images are based upon a `base` image which (in theory) should not have to be rebuilt too often.
+For example; jconsole and Swagger won't work by default out of the box unless `HOST_IP` is defined. In order to build it:
 
 ....
 cd docker/templates/base/latest
 docker build --no-cache -t killbill/base:latest .
 ....
 
+With HOST_IP:
+....
+cd docker/templates/base/latest
+docker build --build-arg HOST_IP=$(docker-machine inspect default -f '{{.Driver.IPAddress}}') -t killbill/base:latest .
```

Comment:
The problem with this approach is that both the base and the killbill images need to be rebuilt, and so for each host.

Questions:

1. Do we really need to change `-i localhost`? I would assume the Ansible inventory is unrelated to Swagger/JConsole
2. Could a similar approach work but with passing `-e ENV_HOST_IP` when starting the image instead?

Reply:
1. Since the `setenv.sh.j2` uses `inventory_hostname`, it will be resolved with what is on `-i localhost`.
2. I suppose it should work. I will need to change the `setenv.sh.j2` and leave `-i localhost` as is, and give it a test.

Reply:
> 2. I suppose it should work. I will need to change the `setenv.sh.j2` and leave `-i localhost` as is, and give it a test.

Yeah, I think changing `inventory_hostname` in `setenv.sh.j2` should be fine.